### ISSUE TITLE
no-empty-file rule seems not support some commonjs modules.

### ISSUE THREAD

**Issue by jason-chang** [sindresorhus/eslint-plugin-unicorn#2201]
<! -- 1️⃣ Explain here what's wrong -->
it seems like that if a file no var definition. will report warning.

![image](https://github.com/sindresorhus/eslint-plugin-unicorn/assets/4367687/41a73287-abd8-4265-b5b7-7a380517f028)

version:  "^48.0.1"
rule: no-empty-file

```js
module.exports = {
  extends: [
    '@lemonbot.fun/eslint-config-base', 
  ],
};
```

**Comment by Mnigos** [comment]
same issue here!

**Comment by fisker** [comment]
Try to uninstall-reinstall your plugin, it's fixed in 28.0.1 https://github.com/sindresorhus/eslint-plugin-unicorn/releases/tag/v48.0.1

**Comment by fisker** [comment]
Try to uninstall-reinstall your plugin, it's fixed in 28.0.1 https://github.com/sindresorhus/eslint-plugin-unicorn/releases/tag/v48.0.1
